packages/eslint-config-eslint/eslintrc.js

```diff
@@ -0,0 +1,451 @@
+/**
+ * @fileoverview the config will be removed in the future, please use the config in the root directory instead.
```

Comment:
Helpful to use the `TODO:` comment prefix for reminders to remove/fix something later